Meeting minutes for March 28, 2025 (#24)
Cardano-Atlantic-Council/Minutes: 20250328.md

### Meeting Minutes

**Cardano Atlantic Council Meeting**

**Date:** March 28, 2025 19:00 UTC

**Attendees:** 
- Beatrice Anihiri
- Thiago Nunes
- Adam Dean
- Jenny Brito
- Richard McCracken

**Purpose:** 

> 1. Discussion on NCL #1 and NCL#2 - two Net Change Limit proposals for 2025 that are both currently on-chain.

Reviewer: ```suggestion
1. Discussion on NCL 1 and NCL 2 - two Net Change Limit proposals for 2025 that are both currently on-chain.
```

**Meeting Points:**
1. **Two NCLs at present**
>What are we going to do if both of the NCL proposals end up passing, which one is in effect for the year 2025?

 - The first NCL proposal gets overwritten by the new NCL proposal.

>  - Article 3 section 3, and appendix 1 section 3 do not clearly define the current circumstance.

Reviewer: ```suggestion
 - Article III Section 3, and Appendix I, Section 3 do not clearly define the current circumstance.
```

 - The 350 million overwrites the 300 million for 2025 if subsequently approved by DReps.
 - The existing one still has 250 million Ada for 2026, what becomes of that if approved?
 - Treat as LIFO, last in first out. The most recently approved INFO NCL by DReps stands. Previous NCL INFO is overwritten entirely.
 - EG: The new proposal kicks out the NCL for 2026 from the first NCL proposal.

>Result: The most recently passed NCL is the only NCL in effect.

**Action Items:**

> 1. Complete rationale for NCL #1 as unconstitutional.
> 2. Cast votes on chain for NCL #1 by Friday, April 4th.
> 3. Internal decision on NCL #2 starting Monday, March 31.
> 4. Cast decision on NCL #2 by Friday, April 4th.

Reviewer: ```suggestion
1. Complete rationale for NCL 1 as unconstitutional.
2. Cast votes on chain for NCL 1 by Friday, April 4th.
3. Internal decision on NCL 2 starting Monday, March 31.
4. Cast decision on NCL 2 by Friday, April 4th.
```
